.openpublishing.publish.config.json

```diff
@@ -259,6 +261,10 @@
     {
       "ReferenceTOC": "docs/docs-ref-autogen/word_1_3/toc.yml",
       "ConceptualTOCUrl": "/office/dev/add-ins/toc.json"
+    },
```

Comment:
Should there also be a similar entry for excel 1.15?

Reply:
Yes! I missed that. Good catch